tests/components/nextcloud/test_config_flow.py

```diff
@@ -0,0 +1,121 @@
+"""Tests for the Nextcloud config flow."""
+from unittest.mock import Mock, patch
+
+from nextcloudmonitor import NextcloudMonitorError
+
+from homeassistant.components.nextcloud import DOMAIN
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+from homeassistant.const import CONF_PASSWORD, CONF_URL, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+
+async def test_form_create_entry(
```

Comment:
Please ensure that` async_setup_entry` is mocked in config flow unit tests
See https://github.com/home-assistant/core/pull/89358